sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -949,9 +949,11 @@ export class MessageReceiver extends LinkEntity {
             }),
           connectionId: connectionId,
           operationType: RetryOperationType.receiverLink,
-          times: Constants.defaultConnectionRetryAttempts,
-          connectionHost: this._context.namespace.config.host,
-          delayInSeconds: 15
+          retryOptions: {
+            maxRetries: Constants.defaultMaxRetries,
```

Comment:
shouldn't this be `Constants.defaultMaxRetriesForConnection`?

Reply:
yes - updated